index.html

```diff
@@ -1334,22 +1336,81 @@ <h2>Characters</h2>
+    <aside class="example" id="family-example" title="Composing Emoji Sequences">
+        <p>Another example of the complex relationship between code points and grapheme clusters are certain emoji. The emoji character for "family" has a code point in Unicode: <span class="codepoint" translate="no"><bdi lang="en">&#x1F46A;</bdi><code class="uname">U+1F46A FAMILY</code></span>. It can also be formed by using using a sequence of code points: <code class="uname">U+1F468 U+200D U+1F469 U+200D U+1F466</code>.</p>
+        
+        <p>This compositional mechanism can be used to create other family variations. For example, the sequence <span class="codepoint" translate="no"><bdi translate="no">&#x1f468;&#x200d;&#x1f469;&#x200d;&#x1f467;&#x200d;&#x1f466;</bdi><code class="uname" translate="no">U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F466</code></span> results in a composed emoji character for a "family: man, woman, girl, boy" on systems that support this kind of composition. That long character sequence still represents just a single [=visual text unit=]. Other characters, such as skin tone modifiers, can further extend the [=grapheme cluster=]:</p>
 
-    <p>A set of rules for converting code points to or from code units is called a <a>character encoding form</a> (or just "character encoding" for short.</p>
+        
+        <table class="cpExample">
+            <tr>
+                <th style="width:10%">Emoji</th>
+                <th>Code Point Sequence</th>
+            </tr>
+            <tr>
+                <td class="bigtext">&#x1F46A;</td>
+                <td><code class="uname">U+1F46A FAMILY</code></td>
+            </tr>
+            <tr>
+                <td class="bigtext">&#x1F468;&#x200d;&#x1f469;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+200D U+1F469 U+200D U+1F466</code></td>
+            </tr>
+             <tr>
+                <td class="bigtext">&#x1F468;&#x200d;&#x1f469;&#x200d;&#x1f467;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F466</code></td>
+            </tr>  
+            <tr>
+                <td class="bigtext">&#x1F468;&#x1f3ff;&#x200d;&#x1f469;&#x1f3fe;&#x200d;&#x1f467;&#x1f3fc;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+1F3FF U+200D U+1F469 U+1F3FE U+200D U+1F467 U+1F3FC U+200D U+1F466</code></td>
+            </tr>
+        </table>
+        
+        
+        <p>Many common emoji can <em>only</em> be formed using sequences of code points, but should be treated as a single [=visual text unit=] when displaying or processing the text. You wouldn't want to put a line-break in the middle of the family!</p>
```

Comment:
The emoji produced in these 4 examples is inconsistent, and the last example has text flowing over the graphics.

Reply:
Your browser must render this differently. On FF/Windows and Chrome/Windows it works fine. My Linux FF does fail here (I suppose I need to install the color emoji font).

After last week's call, I dithered about replacing this example with graphics and that's probably what I should do.

![image](https://github.com/user-attachments/assets/28c3db5f-486b-4e2b-910a-460908a06181)

Reply:
Yes. That's what I got on Linux.